src/components/transactions/eip5792/sendCalls.js

```diff
@@ -148,6 +148,7 @@ export function sendCallsComponent(parentContainer) {
       version: VERSION,
       from,
       chainId,
+      atomicRequired: true,
```

Comment:
LGTM! I guess this will work once the PR for this change is merged too :+1: 

![Screenshot from 2025-04-14 14-32-32](https://github.com/user-attachments/assets/968c8345-fffe-42c7-8622-85bd4ab68c6d)